src/funtest/features/download.feature

```diff
@@ -81,4 +81,24 @@ Feature: Download a Candidate Version
     And the binary for candidate java 8u101 on platform LINUX_64 has a checksum LINUX_64aa605037178b63a using algorithm SHA-1
     When a download request is made on "/download/java/8u101/darwin"
     Then a redirect to "http://download.oracle.com/otn-pub/java/jdk/8u101-b13/jdk-8u101-macosx-x64.dmg" is returned
-    And a checksum "MAC_OSXaa605037178b63a" for header "X-Sdkman-Checksum-SHA-1" is returned
\ No newline at end of file
+    And a checksum "MAC_OSXaa605037178b63a" for header "X-Sdkman-Checksum-SHA-1" is returned
+
+  Scenario: Download a Universal binary with a zip file
```

Comment:
Would you consider phrasing this as _Propagate a content header for a zip file_?

Reply:
done